deployment/constants.py

```diff
@@ -63,4 +64,30 @@
     TAPIR: "https://porter-tapir.nucypher.io/get_ursulas",
 }
 
+#
+# Domain Seednodes
+#
+
+NETWORK_SEEDNODE = {
+    MAINNET: "https://mainnet.nucypher.network:9151/status?json=true",
+    LYNX: "https://lynx.nucypher.network:9151/status?json=true",
+    TAPIR: "https://tapir.nucypher.network:9151/status?json=true",
+}
+
+
+class RITUAL_STATE(IntEnum):
```

Comment:
Since this is a class lets keep the camel case.

```suggestion
class RitualState(IntEnum):
```

Reply:
✅ You're right! d0ca6c7abd228b83a1a475d9fcd7513529bc59af